lib/config/types.ts

```diff
@@ -176,7 +176,7 @@ export interface PostUpgradeTasks {
 }
 
 export type UpdateConfig<
-  T extends RenovateSharedConfig = RenovateSharedConfig
+  T extends RenovateSharedConfig = RenovateSharedConfig,
```

Comment:
we can do the trailing comma change separate via prettier config, so this PR gets much smaller.